tracer/src/Datadog.Trace/Debugger/Instrumentation/Collections/MethodMetadataInfo.cs

```diff
@@ -69,5 +71,37 @@ public MethodMetadataInfo(string[] parameterNames, string[] localVariableNames,
         /// Gets the type that represents the kickoff method of the state machine MoveNext (i.e. original method)
         /// </summary>
         public MethodBase KickoffMethod { get; }
+
+        /// <summary>
+        /// Gets the probe ids actively instrumented
+        /// </summary>
+        public string[] ProbeIds { get; private set; }
+
+        /// <summary>
+        /// Gets the indices of the probes, used to lookup for <see cref="ProbeData"/> from <see cref="ProbeDataCollection"/>.
+        /// </summary>
+        public int[] ProbeMetadataIndices { get; private set; }
+
+        /// <summary>
+        /// Gets the unique sequence of the active instrumentation that is being applied. It basically identifies the probe ids combination that is currently applied.
+        /// If a probe is getting added and/or removed, then this number will be re-updated using new instrumentation, and have a new sequence number, "hard-coded" by the instrumentation.
+        /// </summary>
+        public int InstrumentationSequence { get; private set; } = -1;
```

Comment:
Shouldn't this be called the `InstrumentationVersion` then?

Reply:
Mhmm, I considered naming it `InstrumentationVersion` but decided against it since this number does not represent a "version" of a specific method. Meaning, having a method `Foo` the sequence could be `0` and upon adding/deleting probes it would be `100`. The only guarantee of this number is that it will be "unique" and "incremental". Nothing more, nothing less. Thus, I decided naming it a "sequence" would better represent what it is.
What do you think?

Reply:
I think naming is hard :) And that they're both not ideal, but personally I would prefer `version` over `sequence` - the former at least suggest that is represents the state of things at a certain point in time, while the latter doesn't. A few other alternatives could be `InstrumentationRevision`, `InstrumentationMarker`, `InstrumentationCheckpoint`, but I tend to think `Version` is the easiest to grok, even if you would have to explicitly call out that the values won't necessarily be incremented in arithmetic progression like standard version numbers.

Reply:
I don't have a strong opinion on this so I'll go with `InstrumentationVersion`.

Reply:
Done in d22e27f7934085a341580679dce764df01b41514.